### ISSUE TITLE
Currently Data is storing in App Data inside the C Directory.

### ISSUE THREAD

**Issue by haseeb4091** [drkNsubuga/PharmaSpot#14]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by drkNsubuga** [comment]
Yes, it is. Please provide more details if there is a problem with it.

**Comment by haseeb4091** [comment]
Thanks for your response.
If any pharmacy is using this software.
And there windows got currpted then there data will loss.To prevent this
issue we need to store data in other drive.
Some issues in code that I see
If stock check in unchecked then code cause error due to its undefined.
Some other issues some time transaction is not showing and I clear all data
now it is working fine .
